addons/website/static/src/builder/plugins/options/website_page_config_option_plugin.js

```diff
@@ -45,13 +62,43 @@ class WebsitePageConfigOptionPlugin extends Plugin {
         save_handlers: this.onSave.bind(this),
     };
 
-    getVisibilityItem() {
-        const isHidden = this.document
-            .querySelector("#wrapwrap > header")
-            .classList.contains("o_snippet_invisible");
-        const isOverlay = this.document
-            .getElementById("wrapwrap")
-            .classList.contains("o_header_overlay");
+    /**
+     * Return the HTML element corresponding to the given type.
+     *
+     * @param {'header' | 'breadcrumb'} type The element type to retrieve.
+     * @returns {HTMLElement | null} The matching element or null if not found.
+     */
+    getTarget(type) {
+        if (type === "header") {
+            return this.document.querySelector("#wrapwrap > header");
+        } else if (type === "breadcrumb") {
+            return this.document.querySelector("#wrapwrap div.o_page_breadcrumb");
+        }
+        return null;
+    }
+
+    /**
+     * Get the current visibility state of the target element.
+     *
+     * @param {'header' | 'breadcrumb'} selector The target element type.
+     * @returns {'regular' | 'overTheContent' | 'hidden'} The visibility state.
+     */
+    getVisibilityItem(selector) {
```

Comment:
Instead of selector can we keep something like type or element_type? selector sounds like css selector